core/matchers/src/main/kotlin/au/com/dius/pact/core/matchers/Matchers.kt

```diff
@@ -3,15 +3,15 @@ package au.com.dius.pact.core.matchers
 import au.com.dius.pact.core.matchers.util.corresponds
 import au.com.dius.pact.core.matchers.util.tails
 import au.com.dius.pact.core.model.PathToken
-import au.com.dius.pact.core.model.matchingrules.MatchingRuleGroup
-import au.com.dius.pact.core.model.matchingrules.MatchingRules
+import au.com.dius.pact.core.model.matchingrules.*
```

Comment:
They seem to always use single name imports throughout their code. Let's stay with their convention.

```suggestion
import au.com.dius.pact.core.model.matchingrules.IgnoreOrderMatcher
import au.com.dius.pact.core.model.matchingrules.MatchingRuleGroup
import au.com.dius.pact.core.model.matchingrules.MatchingRules
import au.com.dius.pact.core.model.matchingrules.MaxEqualsIgnoreOrderMatcher
import au.com.dius.pact.core.model.matchingrules.MinEqualsIgnoreOrderMatcher
import au.com.dius.pact.core.model.matchingrules.MinMaxEqualsIgnoreOrderMatcher
```
In intellij, you can set this in preferences, code style, kotlin, imports (tab) and then set everything to single name.